SBOM/spdx.md

```diff
@@ -0,0 +1,216 @@
+                },
+                {
+                    "annotationDate": "2023-06-19T10:49:04.840630Z",
+                    "annotationType": "OTHER",
+                    "annotator": "Tool: alma-sbom 0.0.1",
+                    "comment": "almalinux:sbom:casHash=1bfcf5bf3187ff21db8109d48a0198b92341a9124f63db82cf501d1968ba1374"
+                },
+                {
+                    "annotationDate": "2023-06-19T10:49:04.840659Z",
+                    "annotationType": "OTHER",
+                    "annotator": "Tool: alma-sbom 0.0.1",
+                    "comment": "almalinux:albs:build:ID=5636"
+                },
+                {
+                    "annotationDate": "2023-06-19T10:49:04.840687Z",
+                    "annotationType": "OTHER",
+                    "annotator": "Tool: alma-sbom 0.0.1",
+                    "comment": "almalinux:albs:build:URL=https://build.almalinux.org/build/5636"
+                },
+                {
+                    "annotationDate": "2023-06-19T10:49:04.840715Z",
+                    "annotationType": "OTHER",
+                    "annotator": "Tool: alma-sbom 0.0.1",
+                    "comment": "almalinux:albs:build:author=eabdullin1 <55892454+eabdullin1@users.noreply.github.com>"
+                },
+                {
+                    "annotationDate": "2023-06-19T10:49:04.840743Z",
+                    "annotationType": "OTHER",
+                    "annotator": "Tool: alma-sbom 0.0.1",
+                    "comment": "almalinux:albs:build:source:gitURL=https://git.almalinux.org/rpms/bash.git"
+                },
+                {
+                    "annotationDate": "2023-06-19T10:49:04.840770Z",
+                    "annotationType": "OTHER",
+                    "annotator": "Tool: alma-sbom 0.0.1",
+                    "comment": "almalinux:albs:build:source:type=git"
+                },
+                {
+                    "annotationDate": "2023-06-19T10:49:04.840798Z",
+                    "annotationType": "OTHER",
+                    "annotator": "Tool: alma-sbom 0.0.1",
+                    "comment": "almalinux:albs:build:source:gitCommit=1a3ebc6398c31698863ba9b31a9f5effdb425c4f"
+                },
+                {
+                    "annotationDate": "2023-06-19T10:49:04.840826Z",
+                    "annotationType": "OTHER",
+                    "annotator": "Tool: alma-sbom 0.0.1",
+                    "comment": "almalinux:albs:build:source:gitRef=imports/c9/bash-5.1.8-6.el9_1"
+                },
+                {
+                    "annotationDate": "2023-06-19T10:49:04.840865Z",
+                    "annotationType": "OTHER",
+                    "annotator": "Tool: alma-sbom 0.0.1",
+                    "comment": "almalinux:albs:build:source:gitCommitCasHash=70c5a02df7074eada535aa35643f019574f7005ff345fd6f482eca797ae7c1d8"
+                }
+            ],
+            "builtDate": "2023-01-24T09:44:30.238513+00:00Z",
```

Comment:
`builtDate` should follow `YYYY-MM-DDThh:mm:ssZ` format

Reply:
You're right, I'll fix my PR. I assumed the format was okay because the validator didn't complain about it. I'll check if this is a known issue with spdx-tools and report it upstream if necessary.

For the record:
https://spdx.github.io/spdx-spec/v2.3/package-information/#726-built-date

Reply:
Had the same impression here, as you said, maybe double check if this is something going wrong on `spdx-tools` side. Meanwhile, yes, please do fix your PR to `alma-sbom`

Reply:
There's definitely something wrong with spdx-tools. Even assigning a `datetime.now()` causes the timestamp to be in incorrect format (it includes milliseconds). I have to check if I can enforce the right format from outside spdx-tools.

Reply:
I created an issue upstream [1] and one of the developer fixed it just a few minutes ago [2].

[1] https://github.com/spdx/tools-python/issues/718
[2] https://github.com/spdx/tools-python/pull/719